powershell-adapter/Tests/powershellgroup.resource.tests.ps1

```diff
@@ -128,4 +128,53 @@ Describe 'PowerShell adapter resource tests' {
         $t = $resources | ? {$_.Type -eq 'TestClassResource/TestClassResource'}
         $t.properties | Should -Contain "BaseProperty"
     }
+
+    It 'Verify highest module version is loaded' {
+
+        $srcPath = Join-Path $PSScriptRoot 'TestClassResource'
+        $pathRoot1 = Join-Path $TestDrive 'A'
+        $pathRoot2 = Join-Path $TestDrive 'B'
+        $path1 = Join-Path $pathRoot1 'TestClassResource' '0.0.1'
```

Comment:
Since this is a test resource, it may be better to use `1.0`, `1.1`, `2.0`, `2.0.1` type versions

Reply:
Updated.